sdks/python/apache_beam/ml/inference/sklearn_inference_test.py

```diff
@@ -134,9 +133,9 @@ def test_predict_output(self):
         numpy.array([1, 2, 3]), numpy.array([4, 5, 6]), numpy.array([7, 8, 9])
     ]
     expected_predictions = [
-        api.PredictionResult(numpy.array([1, 2, 3]), 6),
-        api.PredictionResult(numpy.array([4, 5, 6]), 15),
-        api.PredictionResult(numpy.array([7, 8, 9]), 24)
+        base.PredictionResult(numpy.array([1, 2, 3]), 6),
```

Comment:
WDYT of just importing PredictionResult directly like we do in the pytorch test.

Reply:
yep. we can do that.

Reply:
Done